modules/my_health/spec/request/v1/threads_request_spec.rb

```diff
@@ -12,100 +12,146 @@
   let(:inbox_id) { 0 }
   let(:message_id) { 660_516 }
   let(:thread_id) { 660_515 }
+  let(:mhv_account_type) { 'Premium' }
   let(:va_patient) { true }
   let(:current_user) { build(:user, :mhv, va_patient:, mhv_account_type:) }
   let(:inflection_header) { { 'X-Key-Inflection' => 'camel' } }
 
   before do
-    allow(SM::Client).to receive(:new).and_return(authenticated_client)
     sign_in_as(current_user)
   end
 
-  context 'Basic User' do
-    let(:mhv_account_type) { 'Basic' }
-
-    before { patch '/my_health/v1/messaging/threads/7259506/move?folder_id=0' }
-
-    include_examples 'for user account level', message: 'You do not have access to messaging'
-    include_examples 'for non va patient user', authorized: false, message: 'You do not have access to messaging'
-  end
+  context 'when sm session policy is enabled' do
+    before do
+      Flipper.enable(:mhv_sm_session_policy)
```

Comment:
same here, is you allow(Flipper)... you won't have the possibility of running into flaky specs